scripts/build/termux_step_start_build.sh

```diff
@@ -4,6 +4,23 @@ termux_step_start_build() {
 	# Path to hostbuild marker, for use if package has hostbuild step
 	TERMUX_HOSTBUILD_MARKER="$TERMUX_PKG_HOSTBUILD_DIR/TERMUX_BUILT_FOR_$TERMUX_PKG_VERSION"
 
+	# Check if versions are aligned between linked packages.
+	# Also check if there are other packages wanting to keep version aligned with this package.
+	local PACKAGES_WANTING_VERSIONS_ALIGNED="$(grep -R -l -E 'TERMUX_PKG_ALIGN_VERSION_WITH=["'"'"']?'"$TERMUX_PKG_NAME"'["'"'"']?' --include="build.sh" ${TERMUX_SCRIPTDIR} ||:)"
```

Comment:
You gotta make this regex soup more readable.
```suggestion
	local PACKAGES_WANTING_VERSIONS_ALIGNED="$(grep -R -l \
		-E "TERMUX_PKG_ALIGN_VERSION_WITH=['\"]?$TERMUX_PKG_NAME['\"]?" \
		--include="build.sh" ${TERMUX_SCRIPTDIR} || :)"
```
Also this should be an array.